resources/EPvalidator.rb

```diff
@@ -104,12 +103,13 @@ def self.run_validator(hpxml_doc)
       "/HPXML/Building/BuildingDetails/ClimateandRiskZones/WeatherStation" => {
         "SystemIdentifier" => one, # Required by HPXML schema
         "Name" => one, # Required by HPXML schema
-        "[WMO | extension/EPWFileName]" => one_or_more, # Reference weather/data.csv for the list of acceptable WMO station numbers
+        "WMO | extension/EPWFileName" => one_or_more, # Reference weather/data.csv for the list of acceptable WMO station numbers
```

Comment:
I can't figure out why this is `one_or_more`. I know this isn't your change, but let's try changing it to `one` and see if anything complains.

```suggestion
        "WMO | extension/EPWFileName" => one, # Reference weather/data.csv for the list of acceptable WMO station numbers
```